tests/test_permissions.py

```diff
@@ -0,0 +1,319 @@
+from typing import Dict, Literal, Optional
+
+import pytest
+
+from disnake.permissions import PermissionOverwrite, Permissions
+
+
+class TestPermissions:
+    def test_init_permissions_keyword_arguments(self) -> None:
+        perms = Permissions(manage_messages=True)
+
+        assert perms.manage_messages is True
+
+        # check we only have the manage message permission
+        assert perms.value == Permissions.manage_messages.flag
+
+    @pytest.mark.skip("this behavior is currently undefined")
```

Comment:
This could likely be resolved through a note in the docs.
We should test both orders though to ensure the behavior won't change in the future.